modules/apps/permission-template/permission-template-service/service.xml

```diff
@@ -1,6 +1,26 @@
 <?xml version="1.0"?>
 <!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.4.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_4_0.dtd">
 
-<service-builder dependency-injector="ds" package-path="com.liferay.permission.template">
-	<namespace>FOO</namespace>
+<service-builder auto-import-default-references="false" auto-namespace-tables="false" dependency-injector="ds" mvcc-enabled="true" package-path="com.liferay.permission.template">
+	<namespace>PermissionTemplate</namespace>
+	<entity change-tracking-enabled="true" local-service="true" name="PermissionTemplateSetting" remote-service="false">
```

Comment:
After discussion, we will change the name of the entity to `PermissionTemplate`

Reply:
Fixed it. Thanks